readme.md

```diff
@@ -111,6 +111,34 @@ cuts out back and forth of correction had someone else written it.
 See: https://github.com/dwyl/process-handbook/issues/61, linked to #5
 
 
+## Glossary of Key Terms
+
+- **Scrum Master**: tracks team progress, liaises with product owner on
+requirements, lends expertise on managing end to end projects, ensures features
+are tested when delivered and manages deployments in conjunction with technical
+architect and the team.
+- **Product backlog:** a prioritised features / to do list for the product found
+on github. See our ['What is the backlog?' section](#what-is-the-backlog?) for
+more detail.
+- **Sprint Backlog**: the backlog that is aiming for completion within a given
+sprint period.
+- **Sprint**: a 1-3 week working lifecycle of planning, building, presentation and
```

Comment:
I would probably move `Sprint` up above the backlogs (as you refer to a sprint backlog before defining a sprint) but not a blocker!